### PR TITLE
Complete refactoring of main.scss & main pages

### PR DESCRIPTION
**Pull request by thevangelist**
# Debrands Lepo.IO away from Lepovirta, rebrands towards Lepo.IO

* Refactors site to use earlier mentioned indigo color: #4E0082
* Markup cleanups
* CSS cleanups (yet some remain, but I'll continue this)
* Responsiveness
* Font-sizing with REM + px fallback
* Remove most &raquo; elements
* Add temp favicon

## TODO:

* Check unused CSS (use more modern stuff, aye)
* More sensible markup
* Unify layout / colors
* Add logo
* Make slogan more visible
* etc.

### CODE REVIEW

File: resources/assets/styles/main.scss [hunk 1]
```diff
@@ -241,51 +155,65 @@ footer {
...
+      &:hover {
+        color: $header_fg;
+        &:before { content: "{" }
+        &:after { content: "}" }
+      }
     }
   }
 }
 
-.figure {
```
Comment (jkpl) [inline resources/assets/styles/main.scss]: Why was `.figure` removed?

File: resources/assets/styles/main.scss [hunk 2]
```diff
@@ -241,51 +155,65 @@ footer {
...
-  text-align: center;
-  img {
-    border: 1px solid $default_border;
-    max-width: 100%;
+#wrapper {
```
Comment (jkpl) [inline resources/assets/styles/main.scss]: `#wrapper` to `.wrapper` to enable multiple wrapper elements.

File: resources/assets/styles/main.scss [hunk 3]
```diff
@@ -241,51 +155,65 @@ footer {
...
+  width: $content_max_width;
+  margin: 0 auto;
+  #projects {
```
Comment (jkpl) [inline resources/assets/styles/main.scss]: Perhaps `#projects` should be untied from `#wrapper`.

File: resources/templates/base.html [hunk 4]
```diff
@@ -2,37 +2,35 @@
 <html>
 <head>
   <meta charset="utf-8"/>
-  <title>{{site-title}}&raquo; {% block subtitle %}{% endblock %}</title>
+  <title>{{site-title}} - {% block subtitle %}{% endblock %}</title>
```
Comment (jkpl) [inline resources/templates/base.html]: When we figure out a slogan, we could put into the `subtitle` block to show it as a default page subtitle.

**Comment by jkpl** [comment]
Great work! I can tune out the last bits today if you're okay with it. :+1:

File: resources/templates/base.html (hunk 4 above)
Comment (thevangelist) [inline resources/templates/base.html]: Good!

**Comment by thevangelist** [comment]
@jkpl I'll check those couple of issues and perhaps you can rebase against this branch then?

File: resources/assets/styles/main.scss (hunk 2 above)
Comment (thevangelist) [inline resources/assets/styles/main.scss]: Let's change this if we have multiple wrappers :)

File: resources/assets/styles/main.scss (hunk 1 above)
Comment (thevangelist) [inline resources/assets/styles/main.scss]: The styling was okay without it, maybe i'll remove the figure class from HTML as well?

Comment (jkpl) [inline resources/assets/styles/main.scss]: The images aren't centered without `.figure`. See [the current version](https://lepo.io/posts/2015-10-30-solving-problems-in-a-generic-way-using-shapeless.html) with window width >800px. In that page, both images are >600px wide so alignment doesn't affect them in the this version. However, smaller images will be aligned to the left side.

**Pull request merged**